wflambda.go

```diff
@@ -0,0 +1,283 @@
+package wflambda
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"github.com/aws/aws-lambda-go/lambdacontext"
+	"github.com/rcrowley/go-metrics"
+	"github.com/wavefronthq/go-metrics-wavefront"
+	"log"
+	"os"
+	"reflect"
+	"strings"
+	"time"
+)
+
+type lambdaHandlerFunction func(context.Context, json.RawMessage) (interface{}, error)
+
+var (
+	server                  string
+	authToken               string
+	isReportStandardMetrics bool
+	csCounter               metrics.Counter
+	csEventCounter          metrics.Counter
+	invocationsCounter      metrics.Counter
+	invocationEventCounter  metrics.Counter
+	eCounter                metrics.Counter
+	eEventCounter           metrics.Counter
+	dGauge                  metrics.Gauge
+	isColdStart             = true
+)
+
+/* Returns the Wavefront Lambda wrapper. The wrapper collects aws lambda's
```

Comment:
Can you please change the comment to adhere to the godoc format: https://blog.golang.org/godoc-documenting-go-code

i.e use `//` for each comment line instead of within `/*..*/`. This is how it is done within the go source code as well, for example: https://golang.org/src/strings/strings.go